scantools/run_navvis_to_capture.py

```diff
@@ -136,6 +134,17 @@ def run(input_path: Path, capture: Capture, tiles_format: str, session_id: Optio
                     pose = get_pose(nv, upright, frame_id, camera_id, tile_id)
                     trajectory[timestamp_us, sensor_id] = pose
 
+    if export_trace:
+        sensors['trace'] = create_sensor('trace', name='Mapping path')
+        for trace in nv.get_trace():
+            timestamp_us = int(trace["nsecs"]) // 1_000  # convert from ns to us
+            qvec = np.array([trace["ori_w"], trace["ori_x"], trace["ori_y"], trace["ori_z"]], dtype=np.float64)
```

Comment:
Are we sure about the convetion used for this file? (i.e., rig to world vs world to rig). Should the trace "sensor" be added to the rig?

Reply:
The conversion in this file was actually imu to world! I fixed it in the commits now. It should be ready for review again

Reply:
> Should the trace "sensor" be added to the rig?

I was moving everything suck that the trace entry in the trajectory matches to the "navvis_rig" sensor (when we export as rig). So "trace" is the same as "navvis_rig" but I changed so we can filter it. I don't know if this is the best option. Perhaps @MarioGini can comment here.